### DISCUSSION TITLE
Database Tuning

### DISCUSSION THREAD

**Discussion by shumancw** [nextcloud/all-in-one#2946] [Questions] [answered]
Is there a way, through the use of environmental variables, to perform database tuning?

**Comment by szaimen** [comment, marked as answer]
Hi, we recently did some analysis on the topic and came to the conclusion that it is not worth it. Feel free to do you own analysis by following https://github.com/nextcloud/all-in-one/discussions/2640#discussioncomment-6071733

**Reply by shumancw** [reply]
Thanks for the reply! I'll check it out.
